assets/wizards/readerRevenue/index.js

```diff
@@ -119,11 +122,70 @@ class ReaderRevenueWizard extends Component {
 		return data;
 	};
 
+	/**
+	 * Button handler for Salesforce wizard.
+	 */
+	handleSalesforce = async () => {
+		const { wizardApiFetch } = this.props;
+		const { data } = this.state;
+		const { salesforceData } = data;
+		const salesforceIsConnected = !! salesforceData.refresh_token;
+
+		// If Salesforce is already connected, button should reset settings.
+		if ( salesforceIsConnected ) {
+			const defaultSettings = {
+				client_id: '',
+				client_secret: '',
+				access_token: '',
+				refresh_token: '',
+				instance_url: '',
+			};
+
+			this.setState( { data: { ...data, salesforceData: defaultSettings } } );
+			return this.update( 'salesforce', defaultSettings );
+		}
+
+		// Otherwise, attempt to establish a connection with Salesforce.
+		const { client_id, client_secret } = salesforceData;
+
+		if ( client_id && client_secret ) {
+			const loginUrl = `https://login.salesforce.com/services/oauth2/authorize?response_type=code&client_id=${ encodeURIComponent(
+				client_id
+			) }&client_secret=${ encodeURIComponent( client_secret ) }&redirect_uri=${ encodeURIComponent(
+				window.location.href
+			) }`;
```

Comment:
tip: `addQueryArgs` from `@wordpress/url` can be user to create URLs without the need for string interpolation and encoding.

Reply:
Didn't know that, thanks for the hint!